.pre-commit-config.yaml

```diff
@@ -1,9 +1,11 @@
 repos:
-  - repo: https://github.com/ambv/black
```

Comment:
What is the reason behind dropping this?

Reply:
Uhm, for some reason it didn't work for me (there was a dependency clash with one of blacks dependencies.) And as black was listed as part of the poetry development dependencies we can use that black instead!

Reply:
Personally, I would prefer to drop black out of the poetry dev dependencies and just have it in the pre-commit hook, where it is called.